cmd/livekit-cli/egress.go

```diff
@@ -385,14 +426,17 @@ func updateStream(c *cli.Context) error {
 }
 
 func stopEgress(c *cli.Context) error {
-	info, err := egressClient.StopEgress(context.Background(), &livekit.StopEgressRequest{
-		EgressId: c.String("id"),
-	})
-	if err != nil {
-		return err
```

Comment:
What is the consequence of not returning an error in stopEgress. Does returning an error cause the whole cli to exit with an error code? If so, would changing this behavior potentially break some scripts?

Reply:
good point.. I guess we should keep the existing behavior if only a single id is passed in.